### PR TITLE
Fix mentions

### LINKED ISSUE #148: Slack -> matrix pills are broken on develop

### PR DESCRIPTION
**Pull request by Cadair**
Signed Off By: Stuart Mumford (stuart@cadair.com)

fixes #148

### CODE REVIEW

File: lib/substitutions.js [hunk 1]
```diff
@@ -69,9 +69,6 @@ var slackToMatrix = function(body, file) {
         }
     }
 
-    // remove <> from links, e.g:
-    body = body.replace(/<((https?:\/\/)?[^>]+?)>/g, '$1');
```
Comment (Half-Shot) [inline lib/substitutions.js]: oh?

File: lib/substitutions.js [hunk 2]
```diff
@@ -188,19 +184,18 @@ var htmlEscape = function(s) {
...
-        .then(users => {
-            const displaymap = {};
-            users.forEach(user => {
-                if (user && user[0]) {
-                    displaymap[user[0].display_name] = user[0].id.split("_")[2].split(":")[0];
-                }
-            });
-            return displaymap;
-        });
+    //TODO: why is this not getting a list of users anymore?
```
Comment (Half-Shot) [inline lib/substitutions.js]: I don't think this is true anymore?

File: lib/substitutions.js [hunk 3]
```diff
@@ -110,7 +107,7 @@ var getSlackFileUrl = function(file) {
  * @param {Main} main the toplevel main instance
  * @return An object which can be posted as JSON to the Slack API.
  */
-var matrixToSlack = function(event, main) {
+var matrixToSlack = async function(event, main) {
```
Comment (Half-Shot) [inline lib/substitutions.js]: ### var

File: lib/substitutions.js (hunk 1 above)
Comment (Cadair) [reply lib/substitutions.js]: Yeah I added this because I thought its absence was causing problems, but it seems that I introduced this bug instead.

**Pull request merged**